### ISSUE TITLE
babel: nunjucks does not exist

### ISSUE THREAD

**Issue by r33beers** [mozilla/nunjucks#1176]
While npm installing nunjucks@3.1.5, the following error occurs:
```
> nunjucks@3.1.5 build:transpile /Users/rj.beers/dev/x-web-maestro/node_modules/nunjucks
> babel nunjucks --out-dir .

babel:
  nunjucks does not exist
```
3.1.4 installs fine.

npm version used: 6.5.0
node version used: 8.14.0

**Comment by motzko** [comment]
@fdintino caused by 3.1.5 https://github.com/mozilla/nunjucks/commit/6a3872eb29ef43ccd2c2068b88f26a90bc09a492

**Comment by coolmate** [comment]
Same issue for me.

**Comment by fluxsauce** [comment]
Same, this is breaking all sorts of builds, both with older versions (node 6 / npm 2) and with node 8 / npm 6.

**Comment by Uniphix** [comment]
Same, I cannot even update my own software because of this issue.  (node 10.8 / npm 6.5)

**Comment by motzko** [comment]
I was able to get around it by adding 3.1.4 as an explicit dependency in my own project

**Comment by tyler-johnson** [comment]
If anyone is using yarn, I am getting around this for now by adding this to my package.json:

```
"resolutions": {
  "nunjucks": "3.1.4"
}
```

**Comment by fdintino** [comment]
oh my. I'm so sorry, I did a very dumb thing with this release. I'll fix immediately.

**Comment by fdintino** [comment]
I've released 3.1.6 and unpublished 3.1.5.
